### ISSUE TITLE
How do I run this network with my own data? 

### ISSUE THREAD

**Issue by mostafa501** [zhangbowen-wx/Shape-Completion-with-Points-in-the-Shadow#1]
Hey all,
I see that you have a custom dataloader for the solution, but I was curious how to use this network on my own data. For example, say I have a PCD which is already incomplete, I do not have the ground truth for this. I just want to run inference on this data. How would be the best way to progress using this network? Any help on this would be very much appreciated.
Thank you for the contribution !!

**Comment by zhangbowen-wx** [comment]
Sorry for late reply, 
First you need to normalize your incomplete PCD, then you can use the [pretrained_model(without GT viewpoints)](https://drive.google.com/drive/folders/1vGDKk7QjqtIM1QmhKNoo0R5bQrz0rKA7) to do the point cloud completion. Hope that will be helpful to you.

**Comment by ChrysLiang** [comment]
@zhangbowen-wx can please provide a simple script to test on a single incomplete point cloud input file (*.pcd or *.ply)? or would be appreciate to provide a toy *.h5 data file to test the original test_VP.py script file? thanks very much!
